interactive-bots/message_info/thejwzhang/notes.md

```diff
@@ -0,0 +1,19 @@
+# Notes
+
+I ran into a couple of issues in this task, and I have created a PR on the 
+instructions page to clarify some of the issues.
+
+First, when trying to run the bot, I initially only typed in the bot name 
+to the command as follows: `zulip-run-bot message_info --config-file ~/Documents/zuliprc`. 
+After consulting mentors, I found out instead of  message_info it should be the
+full path to the message_info bot.
```

Comment:
This is actually not needed; from reading through your discussion on chat.zulip.org, I believe this issue was caused by not having activated the venv.